### PR TITLE
[frame/im-online] remove network state from heartbeats

### PR DESCRIPTION
**Pull request by melekes**
substrate PR: https://github.com/paritytech/substrate/pull/14251

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2918113

**Review by bkchr — APPROVED** [review]

File: node/core/dispute-coordinator/src/initialized.rs [hunk 1]
```diff
@@ -214,61 +214,62 @@ impl Initialized {
 			gum::trace!(target: LOG_TARGET, "Waiting for message");
 			let mut overlay_db = OverlayedBackend::new(backend);
 			let default_confirm = Box::new(|| Ok(()));
-			let confirm_write =
-				match MuxedMessage::receive(ctx, &mut self.participation_receiver).await? {
-					MuxedMessage::Participation(msg) => {
-						gum::trace!(target: LOG_TARGET, "MuxedMessage::Participation");
-						let ParticipationStatement {
-							session,
+			let confirm_write = match MuxedMessage::receive(ctx, &mut self.participation_receiver)
```
Comment (melekes) [inline node/core/dispute-coordinator/src/initialized.rs:217]: result of running `cargo fmt`

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2920438

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2920497

**Comment by melekes** [comment]
bot rebase

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2920596

**Review by ggwpez — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]
LGTM % https://github.com/paritytech/substrate/pull/14251#discussion_r1214268931

**Comment by melekes** [comment]
bot rebase

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2996823

**Comment by melekes** [comment]
bot rebase

**Comment by bkchr** [comment]
@melekes this pull requested missed to add the `im_online::v1::Migration` to the list of the `UnreleasedMigrations` in Polkadot/Kusama.

**Pull request merged**